store/storetest/channel_store.go

```diff
@@ -1047,14 +1047,16 @@ func testChannelStoreGetAllChannels(t *testing.T, ss store.Store, s SqlSupplier)
 	t1.Name = model.NewId()
 	t1.Email = MakeEmail()
 	t1.Type = model.TEAM_OPEN
-	store.Must(ss.Team().Save(&t1))
+	_, err := ss.Team().Save(&t1)
+	assert.Nil(t, err)
```

Comment:
nit: Probably the equivalent here would be `require.Nil` instead of `assert.Nil`. For this kind of cases where we need that to go right, we use require to stop the test execution there, because after that the rest of the test make no sense because is inconsistent.

Reply:
This is a general advice for the whole PR.